pkg/external_repos/pulp_external_repos/pulp.go

```diff
@@ -1,4 +1,4 @@
-package external_repos
+package pulp_external_repos
```

Comment:
I moved this to a new package. I don't have a good solution to the cyclic import here. But I think this command is kinda temporary anyway?

Reply:
agreed, i think this is fine.  We'll likely delete this file as part of the integration work